### PR TITLE
docs: some grammar fixes for meshDrawer2D

### PR DESCRIPTION
**Pull request by NathanX-S**

### CODE REVIEW

File: panda/src/grutil/meshDrawer2D.h [hunk 1]
```diff
@@ -39,8 +39,8 @@
 #include "nodePath.h"
 
 /**
- * This class allows the drawing of 2d objects - mainly based on quads and
- * rectangles.  Allows clipping and serverl high level UI theme functions.
+ * This class allows the drawing of 2D objects - mainly based on quads and
+ * rectangles. It allows clipping and serveral high level UI theme functions.
```
Comment (rdb) [inline panda/src/grutil/meshDrawer2D.h:43]: serveral→several?

Comment (NathanX-S) [reply panda/src/grutil/meshDrawer2D.h:43]: yes?

Comment (NathanX-S) [reply panda/src/grutil/meshDrawer2D.h:43]: oh wait no

Comment (NathanX-S) [reply panda/src/grutil/meshDrawer2D.h:43]: oh wait no

**Pull request closed without merging**